cmd/edit.go

```diff
@@ -138,6 +155,15 @@ var EditSecretCmd = &cobra.Command{
 
 			utils.Check(err)
 
+			if !secret.Editable() {
+				notice := []byte(`
+# DANGER! Secrets cannot be updated, only replaced. Once the change is applied, the old values will be lost forever.
+# Note: You can exit without saving to skip.
+			
```

Comment:
Same comment about spacing. Also, since this message is the same as above, using a constant is probably a better idea